satis-config/govcms-security_updates.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "govcms/satis-stable",
```

Comment:
doesn't this need to be govcms/satis-security_updates?

Reply:
it's just a label. User points to satis.govcms.gov.au/security_updates
sending through a fix